Assignment6/RearrangingCars.py

```diff
@@ -0,0 +1,71 @@
+class Parking:
+
+    def __init__(self, cars):
+        self._cars = cars
+        self._size = len(cars)
+
+    def cars(self):
+        return self._cars.copy()
+
+    def size(self):
+        return self._size

```

Comment:
Siim said that it is better :) 
Actually it is to make them unreachable to a user as we don't want him to change them accidentally or not.